File: baekjoon_all/10000+/boj_10813.cpp
// Solve 2023-02-21
// Update 2023-10-10

#include <bits/stdc++.h>
using namespace std;

#define FASTIO ios_base::sync_with_stdio(false);cin.tie(NULL); // boj_15552.cpp
#define SETPRECISION(n) cout << fixed;cout.precision(n); // boj_1008.cpp
#define SIZE(v) (int)v.size()
#define ALL(v) v.begin(),v.end()
using ll = long long;

int main() {
    FASTIO;

    int n, m;
    cin >> n >> m;

    int baskets[101];

    for (int i = 1; i <= n; i++) {
        baskets[i] = i;
    }

    for (int i = 0; i < m; i++) {
        int u, v;
        cin >> u >> v;

        swap(baskets[u], baskets[v]);
    }

    for (int i = 1; i <= n; i++) {
        cout << baskets[i] << ' ';
    }

    cout << '\n';

    return 0;
}

File: baekjoon_all/10000+/boj_10813.py
# Solve 2023-02-23
# Update 2023-10-10

import sys

input = lambda : sys.stdin.readline().rstrip()

n, m = map(int, input().split())
baskets = [i for i in range(n + 1)]

for _ in range(m):
    i, j = map(int, input().split())

    baskets[i], baskets[j] = baskets[j], baskets[i]

print(*baskets[1:])
